third-party/llvm/find-gcc-prefix.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+
+# If building LLVM and Clang with GCC, the built
+# LLVM should use C++ headers from the selected GCC.
+
+# Using the LLVM project's libc++ would be a reasonable alternative.
+
+CC=$1
+WHICH=`which $CC`
+DIR=${WHICH%%/bin/$CC}
+
+if [ -d $DIR/include ]
+then
+  echo $DIR
+else
+  if [ -d $DIR/snos/include ]
+  then
+    echo $DIR/snos
+  else
+    echo "Could not find GCC prefix directory" 1>&1
```

Comment:
What does `1>&1` do? 

Should we be emitting some error code here and not setting `-DGCC_INSTALL_PREFIX` if the script failed?

Reply:
`1>&1` is supposed to be `1>&2` to output to stderr. I'll fix that and make it not set -DGCC_INSTALL_PREFIX if the script doesn't output.